src/ai/susi/DAO.java

```diff
@@ -889,23 +889,21 @@ public static void pushCommit(Git git, String commit_message, String userEmail)
     private static String getConflictsMailContent(MergeResult mergeResult) throws APIException {
         // get template file
         String result="";
-        String conflictLines= "";
+        StringBuilder conflictLines= new StringBuilder();
         try {
             result = IO.readFileCached(Paths.get(DAO.conf_dir + "/templates/conflicts-mail.txt"));
             for (String path :mergeResult.getConflicts().keySet()) {
                 int[][] c = mergeResult.getConflicts().get(path);
-                conflictLines+= "Conflicts in file " + path + "\n";
+                conflictLines.append("Conflicts in file ").append(path).append("\n");
                 for (int i = 0; i < c.length; ++i) {
-                    conflictLines+= "  Conflict #" + i+1 +"\n";
+                    conflictLines.append("  Conflict #").append(i).append(1).append("\n");
```

Comment:
appending i and 1 is not the same as computing the sum of i and 1

Reply:
Thanks for letting me know !
I'll modify it ! :)